frontend/src/components/application/OpportunityCard.tsx

```diff
@@ -0,0 +1,178 @@
+  }
+
+  return null;
+};
+
+const displayAgencyAndCode = (
+  agencyName: string | null,
+  agencyCode: string | null,
+) => (agencyCode && agencyName ? `${agencyName} - ${agencyCode}` : null);
+
+const displayDate = (date: string | null) =>
+  date ? getConfiguredDayJs()(date).format("MMM D, YYYY hh:mm A z") : null;
+
+const OpportunityOverview = ({ opportunity }: OpportunityOverviewProps) => {
+  const t = useTranslations("Application.opportunityOverview");
+  const {
+    agency_code,
+    agency_name,
+    opportunity_assistance_listings,
+    opportunity_id,
+    opportunity_title,
+    opportunity_number,
+    summary
+  } = opportunity;
+
+  return (
+    <>
+      <Grid tablet={{ col: 6 }} mobile={{ col: 12 }}>
+        <dl className="margin-top-0">
+          <OpportunityItemLink
+            opKey={t("name")}
+            opValue={opportunity_title}
+            opId={opportunity_id}
+          />
+          <OpportunityItem opKey={t("number")} opValue={opportunity_number} />
+          <OpportunityItem
+            opKey={t("posted")}
+            opValue={displayDate(summary.post_date)}
+          />
+          <OpportunityItem
+            opKey={t("agency")}
+            opValue={displayAgencyAndCode(agency_name, agency_code)}
+          />
+          <OpportunityItem
+            opKey={t("assistanceListings")}
+            opValue={displayOpportunityAssistanceListings(
+              opportunity_assistance_listings,
+            )}
+          />
+          <OpportunityItem
+            opKey={t("costSharingOrMatchingRequirement")}
+            opValue={displayIsCostSharing(summary.is_cost_sharing)}
+          />
+          <OpportunityItem
+            opKey={t("grantorContactInfomation")}
+            opValue={summary.agency_contact_description}
+          />
+        </dl>
+      </Grid>
+      <Grid tablet={{ col: 6 }} mobile={{ col: 12 }}>
+        <h4 className="font-ui-md text-bold margin-bottom-1">{t("award")}</h4>
+        <dl className="margin-top-0">
+          <OpportunityItem
+            opKey={t("programFunding")}
+            opValue={formatCurrency(summary.estimated_total_program_funding)}
+          />
+          <OpportunityItem
+            opKey={t("expectedAward")}
+            opValue={
+              summary.expected_number_of_awards
+                ? String(summary.expected_number_of_awards)
+                : null
+            }
+          />
+          <OpportunityItem
+            opKey={t("awardMaximum")}
+            opValue={formatCurrency(summary.award_ceiling)}
+          />
+          <OpportunityItem
+            opKey={t("awardMinimum")}
+            opValue={formatCurrency(summary.award_floor)}
+          />
+          <OpportunityItem
+            opKey={t("estimatedAwardDate")}
+            opValue={displayDate(summary.close_date)}
+          />
+        </dl>
+      </Grid>
+    </>
+  );
+};
+
+export const OpportunityCard = (
+  opportunityOverview: OpportunityOverviewType,
```

Comment:
a final nit, since this is a component, the argument you're passing in here is `props`. Once we use this in the wild it will be need to be structured like

```suggestion
export const OpportunityCard = ({ opportunityOverview } : { opportunityOverview: OpportunityOverviewType }) =>
```
but again, it will likely also be easiest if this just takes in the entire opportunity